src/components/Calculator.js

```diff
@@ -1,70 +1,64 @@
-            <button onClick={this.dispspalyValue} type="button">8</button>
-            <button onClick={this.dispspalyValue} type="button">9</button>
-            <button onClick={this.dispspalyValue} type="button" className="signs">x</button>
-            <button onClick={this.dispspalyValue} type="button">4</button>
-            <button onClick={this.dispspalyValue} type="button">5</button>
-            <button onClick={this.dispspalyValue} type="button">6</button>
-            <button onClick={this.dispspalyValue} type="button" className="signs">-</button>
-            <button onClick={this.dispspalyValue} type="button">1</button>
-            <button onClick={this.dispspalyValue} type="button">2</button>
-            <button onClick={this.dispspalyValue} type="button">3</button>
-            <button onClick={this.dispspalyValue} type="button" className="signs">+</button>
-            <button onClick={this.dispspalyValue} type="button" className="item0">0</button>
-            <button onClick={this.dispspalyValue} type="button">.</button>
-            <button onClick={this.dispspalyValue} type="button" className="signs">=</button>
-          </div>
+  if (next) {
+    result = `${next}`;
+  }
+  return (
+    <div className="container">
+      <div className="calc-content">
+        <div className="calc-screen">
+          <div className="op">{op}</div>
+          {result || '0'}
+        </div>
+        <div className="calc-btns">
+          <button onClick={dispspalyValue} type="button">AC</button>
+          <button onClick={dispspalyValue} type="button">+/-</button>
+          <button onClick={dispspalyValue} type="button">%</button>
+          <button onClick={dispspalyValue} type="button" className="signs">&divide;</button>
+          <button onClick={dispspalyValue} type="button">7</button>
+          <button onClick={dispspalyValue} type="button">8</button>
+          <button onClick={dispspalyValue} type="button">9</button>
+          <button onClick={dispspalyValue} type="button" className="signs">x</button>
+          <button onClick={dispspalyValue} type="button">4</button>
+          <button onClick={dispspalyValue} type="button">5</button>
+          <button onClick={dispspalyValue} type="button">6</button>
+          <button onClick={dispspalyValue} type="button" className="signs">-</button>
+          <button onClick={dispspalyValue} type="button">1</button>
+          <button onClick={dispspalyValue} type="button">2</button>
+          <button onClick={dispspalyValue} type="button">3</button>
+          <button onClick={dispspalyValue} type="button" className="signs">+</button>
+          <button onClick={dispspalyValue} type="button" className="item0">0</button>
+          <button onClick={dispspalyValue} type="button">.</button>
+          <button onClick={dispspalyValue} type="button" className="signs">=</button>
         </div>
       </div>
-    );
-  }
-}
+    </div>
+  );
+};
```

Comment:
Refactored from a class-based component to a functional component using [hooks](https://reactjs.org/docs/hooks-intro.html) ✔ Well done!